x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authz/interceptor/BulkShardRequestInterceptor.java

```diff
@@ -34,16 +36,18 @@
 
     private final ThreadContext threadContext;
     private final XPackLicenseState licenseState;
+    private final Settings settings;
 
-    public BulkShardRequestInterceptor(ThreadPool threadPool, XPackLicenseState licenseState) {
+    public BulkShardRequestInterceptor(ThreadPool threadPool, XPackLicenseState licenseState, Settings settings) {
         this.threadContext = threadPool.getThreadContext();
         this.licenseState = licenseState;
+        this.settings = settings;
     }
 
     @Override
     public void intercept(RequestInfo requestInfo, AuthorizationEngine authzEngine, AuthorizationInfo authorizationInfo,
                           ActionListener<Void> listener) {
-        boolean shouldIntercept = licenseState.isSecurityEnabled();
+        boolean shouldIntercept = XPackSettings.SECURITY_ENABLED.get(settings);
```

Comment:
Likewise, I don't think this is needed anymore.

Reply:
Thanks, it's obvious now that I'm looking at it. I guess at some point I started applying the change mechanically. I 'll take care of this in all offending places